### PR TITLE
Add Windows support for salt-master

### PR DESCRIPTION
**Pull request by skizunov**
Add Windows support for salt-master

### CODE REVIEW

**Comment by thatch45** [comment]
Good land! AMAZING!!!!!

**Comment by jfindlay** [comment]
@skizunov, you get the awesome label.

File: salt/utils/network.py [hunk 1]
```diff
@@ -1053,6 +1053,44 @@ def _freebsd_remotes_on(port, which_end):
...
+    Parses output of shell 'netstat' to get connections
+
+    C:\>netstat -n
+
+    Active Connections
+
+       Proto  Local Address          Foreign Address        State
+       TCP    10.2.33.17:3007        130.164.12.233:10123   ESTABLISHED
+       TCP    10.2.33.17:3389        130.164.30.5:10378     ESTABLISHED
+       '''
```
Comment (rallytime) [inline salt/utils/network.py]: Can you move these quotes back in an indentation?

File: salt/utils/network.py [hunk 2]
```diff
@@ -1053,6 +1053,44 @@ def _freebsd_remotes_on(port, which_end):
...
 
+def _windows_remotes_on(port, which_end):
+    '''
+    Windows specific helper function.
+    Returns set of ipv4 host addresses of remote established connections
+    on local or remote tcp port.
+
+    Parses output of shell 'netstat' to get connections
+
+    C:\>netstat -n
```
Comment (rallytime) [inline salt/utils/network.py]: I think by throwing a pylint ignore on here should clean up the pylint error:
```
C:\>netstat -n  # pylint: disable=W1401
```

File: salt/utils/network.py (hunk 1 above)
Comment (thatch45) [inline salt/utils/network.py]: I got it

File: salt/utils/network.py (hunk 2 above)
Comment (thatch45) [inline salt/utils/network.py]: I got it

**Pull request merged**